v4l2/v4l2_encode.cpp

```diff
@@ -58,6 +58,14 @@ bool V4l2Encoder::start()
     streamFormat.streamFormat = AVC_STREAM_FORMAT_ANNEXB;
     m_encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 
+    if (m_svct) {
+        VideoTemporalLayers layers;
+        layers.numLayersMinus1 = 1; // It is two layers actually
+        layers.bitRate[0] = 3000000;
+        layers.bitRate[1] = 4000000;
```

Comment:
The highest layer's bitrate should been set via "VideoParamsCommon.rcParams.bitRate"; So this line should be replaced with:
VideoParamsCommon.rcParams.bitRate = 4000000;